File: app/src/main/java/com/dev/bins/note/adapter/Holder.java
package com.dev.bins.note.adapter;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import com.balysv.materialripple.MaterialRippleLayout;
import com.dev.bins.note.R;

/**
 * Created by bin on 11/26/15.
 */
public class Holder extends RecyclerView.ViewHolder {

    TextView title;
    TextView content;
    TextView time;
    MaterialRippleLayout cardView;

    public Holder(final View itemView) {
        super(itemView);
        title = (TextView) itemView.findViewById(R.id.title);
        content = (TextView) itemView.findViewById(R.id.content);
        time = (TextView) itemView.findViewById(R.id.time);
        cardView = (MaterialRippleLayout) itemView.findViewById(R.id.cardView);
    }

}

File: app/src/main/java/com/dev/bins/note/adapter/Holder.kt
package com.dev.bins.note.adapter

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.TextView

import com.balysv.materialripple.MaterialRippleLayout
import com.dev.bins.note.R

/**
 * Created by bin on 11/26/15.
 */
class Holder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    internal var title: TextView
    internal var content: TextView
    internal var time: TextView
    internal var cardView: MaterialRippleLayout

    init {
        title = itemView.findViewById(R.id.title) as TextView
        content = itemView.findViewById(R.id.content) as TextView
        time = itemView.findViewById(R.id.time) as TextView
        cardView = itemView.findViewById(R.id.cardView) as MaterialRippleLayout
    }

}
